app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationActivity.kt

```diff
@@ -118,13 +120,17 @@ class KeyGearValuationActivity : BaseActivity(R.layout.activity_key_gear_valuati
             val text = priceInput.getText()
             setButtonState(text.isNotEmpty(), date != null)
             if (!text.isNullOrBlank()) {
-                val value = text.toDouble()
-                if (value > maxInsurableAmount.toDouble()) {
-                    animateDateDown()
-                    noCoverage.show()
-                } else {
-                    animateDateUp()
-                    noCoverage.remove()
+                try {
+                    val value = text.toDouble()
+                    if (value > maxInsurableAmount.toDouble()) {
+                        animateDateDown()
+                        noCoverage.show()
+                    } else {
+                        animateDateUp()
+                        noCoverage.remove()
+                    }
+                } catch (e: Exception) {
+                    Timber.e("Cant convert to double")
```

Comment:
It's ok to swallow this exception I think 🙂

Reply:
What?? 😕